MagneticField/GeomBuilder/test/python/testMagGeometryOldDD.py

```diff
@@ -20,32 +20,8 @@
 
 process.load("FWCore.MessageLogger.MessageLogger_cfi")
 
-process.MessageLogger = cms.Service("MessageLogger",
```

Comment:
Ditto - this test is meant to be run only and explicitly to produce this message logger output.